client/conf/configuration.properties

```diff
@@ -2,6 +2,6 @@
 # oxTrus API integration test application configuration file
```

Comment:
We should use `profiles` here to allow change test data without updating file.
In oxTrust/server we uses this approach. You can use it as reference